<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SignFilter</class>
    <widget class="QWidget" name="RoadsSelector">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>200</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Select Roads Layer</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>381</width>
                    <height>180</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Roads Layer</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QComboBox" name="roadsLayerComboBox" />
                </item>
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Road Name Field</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QComboBox" name="fieldNameComboBox" />
                </item>
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Road PK Field</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QComboBox" name="fieldPkComboBox" />
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="pushButton_ok">
                                <property name="text">
                                    <string>OK</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButton_cancel">
                                <property name="text">
                                    <string>Cancel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>